apps/sms/style/attachment.css

```diff
@@ -97,7 +118,7 @@
   position: absolute;
   bottom: 0.5rem;
   right: 0;
-  left: 0;
+  left: 1px;  /* Avoid issues related to being too much on the left */
```

Comment:
I added it by myself but I'm going to drop it, as this should be requested by designers as you say.